components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -284,6 +284,10 @@ func (backend *ESClient) UpdateReportProjectsTags(ctx context.Context, projectTa
 						}
 					}
 					if (condition.roles.length > 0) {
+						if (ctx._source['roles'] == null) {
```

Comment:
The condition does not match if its type is roles and there are no roles on the report.